test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,34 @@
+#include "TutorialServo.hpp"
+#include "mbed.h"
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees, float
+  minPulsewidthInMs, float maxPulsewidthInMs) :
+  m_servoPwmOut(servoPin),
+  m_servoRangeInDegrees(servoRangeInDegrees),
+  m_minPulsewidthInMs(minPulsewidthInMs),
+  m_maxPulsewidthInMs(maxPulsewidthInMs)
+{}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+
+  m_servoPwmOut.period_ms(20);
+  
+  m_servoPwmOut.pulsewidth((m_minPulsewidthInMs + degrees/m_servoRangeInDegrees) / 1000);
```

Comment:
Consider how `m_maxPulsewidthInMs` might affect this mapping

Reply:
Also consider what if degrees was negative or greater than `m_servoRangeInDegrees`?